controllers/user.js

```diff
@@ -131,4 +136,93 @@ export default class UserController {
       });
     });
   }
+
+  /**
+   * @description reset user password
+   * @param {object} req http request object
+   * @param {object} res http response object
+   * @returns {object} response
+   */
+  static async requestpasswordreset(req, res) {
```

Comment:
@youngestdj this should be `requestPasswordReset`

Reply:
I have implemented changes